core/cmd/remote_client.go

```diff
@@ -463,6 +464,36 @@ func (cli *Client) ConfigDump(c *clipkg.Context) (err error) {
 	return nil
 }
 
+func (cli *Client) ConfigFileValidate(c *clipkg.Context) error {
+	fileName := c.String("file")
+	err := validateConfigFile(fileName)
+	if err != nil {
+		fmt.Printf("Validation failed (%s)\n", err.Error())
+	} else {
+		fmt.Println("Validation succeeded!")
```

Comment:
Perhaps at this point we could just `(g *generalConfig) LogConfiguration`, and then it would be implicit that it is valid?